core/scripts/run-tests.sh

```diff
@@ -47,6 +47,9 @@
  *              Specify the path and the extension
  *              (i.e. 'core/modules/user/user.test').
  *
+ *
+ *  --directory Run all tests found within the specified file directory.
```

Comment:
This should match the `--xml` option where `<path>` is on the same line and then the description is given it's own line.